assets/src/design-system/components/button/stories/index.js

```diff
@@ -149,12 +156,61 @@ const ButtonCombosToDisplay = () => (
   </Container>
 );
 
-export const _default = () => {
+export const DarkTheme = () => {
   return (
     <ThemeProvider theme={theme}>
       <ButtonCombosToDisplay />
     </ThemeProvider>
   );
 };
 
-export const LightThemeButtons = () => <ButtonCombosToDisplay />;
+export const LightTheme = () => <ButtonCombosToDisplay />;
+
+const TOGGLE_VARIANTS = [BUTTON_VARIANTS.CIRCLE, BUTTON_VARIANTS.SQUARE];
+const ToggleButtonContainer = ({ isToggled, swapToggled }) => (
+  <Container>
+    {Object.values(BUTTON_SIZES).map((buttonSize) => (
+      <Row key={`${buttonSize}_row_storybook`}>
+        {TOGGLE_VARIANTS.map((buttonVariant) => (
+          <div key={`${buttonVariant}_${buttonSize}_storybook`}>
+            <ToggleButton
+              key={`${buttonVariant}_storybook`}
+              variant={buttonVariant}
+              size={buttonSize}
+              isToggled={isToggled}
+              onClick={swapToggled}
+            >
+              <ButtonContent variant={buttonVariant} />
+            </ToggleButton>
+            <Text>
+              {`variant: ${buttonVariant}`} <br />
+              {`size: ${buttonSize}`} <br />
+              {`is on: ${isToggled}`}
+            </Text>
+          </div>
+        ))}
+      </Row>
+    ))}
+  </Container>
+);
+
+ToggleButtonContainer.propTypes = {
+  isToggled: PropTypes.bool.isRequired,
+  swapToggled: PropTypes.func.isRequired,
+};
+
+export const ToggleButtons = () => {
+  const [isToggled, setToggled] = useState(false);
+  const swapToggled = useCallback(() => setToggled((b) => !b), []);
```

Comment:
This toggles all the buttons at the same time like this -- is this intended (or perhaps just not that important)?

Reply:
Didn't feel it was that important to fix in the demo 🤷